ragaai_catalyst/tracers/utils/rag_trace_json_converter.py

```
@@ -0,0 +1,197 @@
+            for span in input_trace:
+                if span["name"] in ["ChatOpenAI", "ChatAnthropic", "ChatGoogleGenerativeAI"]:
+                    return span["attributes"].get("llm.input_messages.1.message.content")
+
+                elif span["name"] == "LLMChain":
+                    return json.loads(span["attributes"].get("input.value", "{}")).get("question")
+
+                elif span["name"] == "RetrievalQA":
+                    return span["attributes"].get("input.value")
+                
+                elif span["name"] == "VectorStoreRetriever":
+                    return span["attributes"].get("input.value")
```

Ruff (0.8.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)